common/flags.mk

```diff
@@ -0,0 +1,17 @@
+# This makefile is just an easy way to standardise the compiler flags we're using
+# between different samples/mini-libraries.
+#
+# Just put "-include ../common/flags.mk" in your makefile.
+
+CPP_STANDARD = c++14
```

Comment:
I get super-confused  about the different sorts of equals in Makefiles but maybe this should be a ?= so you can set it and then include this. Also maybe add a comment that this is GeNN magic!

Reply:
:+1: